lib/xnvme_be_fbsd_dev.c

```diff
@@ -9,9 +9,6 @@
 #include <fcntl.h>
 #include <dirent.h>
 #include <unistd.h>
-#include <libxnvme_ident.h>
```

Comment:
This should include libxnvme.h

Reply:
Since the internal headers still include other headers, then it gets ``libxnvme.h`` via ``xnvme_be.h``